composer.json

```diff
@@ -36,6 +36,11 @@
       "Localheinz\\OrganizingTestCodeInPhp\\": "src/"
     }
   },
+  "autoload-dev": {
+    "psr-4": {
+      "Localheinz\\OrganizingTestCodeInPhp\\Test\\": "test/"
+    }
+  },
```

Comment:
When you declare test code in separate directories with separate configuration files, managing exclusions for the `composer` autoloader in `composer.json ` is unnecessary. Instead, you configure and document namespaces for test code via an autoloader configuration in the `autoload-dev` section.